crates/rover-client/src/query/service/delete.rs

```diff
@@ -0,0 +1,168 @@
+use crate::blocking::Client;
+use crate::RoverClientError;
+use graphql_client::*;
+
+#[derive(GraphQLQuery)]
+// The paths are relative to the directory where your `Cargo.toml` is located.
+// Both json and the GraphQL schema language are supported as sources for the schema
+#[graphql(
+    query_path = "src/query/service/delete.graphql",
+    schema_path = "schema.graphql",
+    response_derives = "PartialEq, Debug, Serialize, Deserialize",
+    deprecated = "warn"
+)]
+/// This struct is used to generate the module containing `Variables` and
+/// `ResponseData` structs.
+/// Snake case of this name is the mod name. i.e. delete_service_mutation
+pub struct DeleteServiceMutation;
+type RawMutationResponse = delete_service_mutation::DeleteServiceMutationServiceRemoveImplementingServiceAndTriggerComposition;
+
+#[derive(Debug, PartialEq)]
```

Comment:
would love to see some docs on this struct

(sidenote, do we have deny missing docs enabled in `rover-client`?)

Reply:
It would appear not :)